src/components/common/Loading.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+const loadingImg =
+  'https://cdn.auth0.com/blog/auth0-react-sample/assets/loading.svg';
```

Comment:
keep this as is for now, but in the future I'd love to meet with Swapnali and the Designers to get our very own fancy spinner that doesn't rely on `auth0` CDN to be delivered across the network.